go.mod

```diff
@@ -6,8 +6,8 @@ require (
 	github.com/AlecAivazis/survey/v2 v2.3.6
 	github.com/OneOfOne/xxhash v1.2.8
 	github.com/Pallinder/go-randomdata v1.2.0
-	github.com/akitasoftware/akita-ir v0.0.0-20240815175436-58df31060c47
-	github.com/akitasoftware/akita-libs v0.0.0-20240820205141-dc8c30c29a5b
+	github.com/akitasoftware/akita-ir v0.0.0-20241008154243-b6c9139277d2
+	github.com/akitasoftware/akita-libs v0.0.0-20241008155148-e576f410f201
```

Comment:
Please remember to update this after you merge postmanlabs/observability-ir#15 and postmanlabs/observability-shared-libs#217.